build.gradle

```diff
@@ -83,6 +88,26 @@ dependencies {
     testImplementation 'org.openjdk.jmh:jmh-generator-annprocess:1.21'
 }
 
+shadowJar {
+    minimize()
+    archiveClassifier.set('')
+    configurations = [project.configurations.compileClasspath]
+    relocate('com.google.common', 'graphql.com.google.common') {
+        include 'com.google.common.collect.*'
```

Comment:
didnt need base??

Reply:
still working on it: currently whole guava is added to jar. There is no obvious way to split guava itself up.